src/patternfly/components/ClipboardCopy/examples/clipboard-copy-expansion-example.hbs

```diff
@@ -0,0 +1,24 @@
+{{#> clipboard-copy clipboard-copy--id="3"}}
```

Comment:
can you fix the indentation in this file? If you can't see it in vscode, I select all, hit `shift + tab` until everything is flush-left, then just manually re-indent. There is probably a better way to do that though :)